lib/xcribe/config.ex

```diff
@@ -1,121 +1,35 @@
 defmodule Xcribe.Config do
   @moduledoc false
 
-  alias Xcribe.{MissingInformationSource, UnknownFormat}
-
   @valid_formats [:api_blueprint, :swagger]
 
-  @doc """
-  Return true if serve mode is enabled.
-
-  If no config was given the default is false.
-
-  To configure server mode:
-
-      config :xcribe, [
-        serve: true
-      ]
-  """
```

Comment:
Why were all of these functions' docstrings removed? :cry:

Reply:
Because we changed a bit the private functions :P and that doc seems to not be more useful.